guide/sections/part3/sensor-centres.adoc

```diff
@@ -15,34 +15,46 @@ Beyond WIS 2.0 itself, a Sensor Centre relies on WIS 2.0 Global Services and Nod
 
 ==== 3.3.2 Sensor Centres for Global Brokers (SCGB)
 
+===== 3.3.2.1 Technical considerations
+
+* As a convention, the Sensor Centre for Global Broker centre-id will be ``{tld}-{centre-name}-sensor-centre-global-broker``.
+
```

Comment:
* The SCGB will connect to all Global Brokers.
* The SCGB subscribes to ``origin/a/wis2/#, cache/a/wis2/#, monitor/a/wis2/#`` on all GBs
* When the SCGB is successfully connected to the GB, the value of ``wmo_wis2_scgb_connected_flag`` will be set to ``1``. When the SCGB cannot connect to the GB, the value of ``wmo_wis2_scgb_connected_flag`` will be set to ``0``.
* When a message is received on a subscription, the SCGB will increment the metric ``wmo_wis2_scgb_messages_received_total`` by ``1`` and will set the metric ``wmo_wis2_scgb_last_message_timestamp_seconds`` to the current timestamp in seconds.